manifests/f/Funkin'Team/FridayNightFunkin'/v2.8/Funkin'Team.FridayNightFunkin'.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Funkin'Team.FridayNightFunkin'
+PackageVersion: v2.8
+PackageLocale: en-US
+Publisher: Funkin' Team
+PackageName: Friday Night Funkin'
+License: none
```

Comment:
`none` isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, Freeware, or other terms as appropriate